### PR TITLE
NavBar

### PR DESCRIPTION
**Pull request by D3TRU04**

### CODE REVIEW

**Comment by D3TRU04** [comment]
Screenshot 2024-05-30 at 4 23 19 PM

**Comment by D3TRU04** [comment]
I'm almost done but the current issue I have is having the company logo show up on the top left.

File: src/layouts/Layout.astro [hunk 1]
```diff
@@ -1,22 +1,15 @@
...
-		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
-		<meta name="generator" content={Astro.generator} />
-		<title>{title}</title>
-	</head>
-	<body>
-		<slot />
-	</body>
+<html>
+  <head>
+    <title>My Astro Site</title>
```
Comment (txst54) [inline src/layouts/Layout.astro]: Look at the original layout code: get the title from Astro.props and use that, not My Astro Site.

File: src/components/NavBar.astro [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    flex-grow: 1;
+    margin-right: 12rem; 
+  }
+  .nav-container {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+  }
+</style>
```
Comment (txst54) [inline src/components/NavBar.astro]: Use Tailwind for styling, don't use CSS. It is way easier and I am trying to maintain a standard style. Also highlight which page the user is on in the navbar
image

**Comment by txst54** [comment]
For the image, can you push it to the github repo? 
Also, I noticed you are "adding files by upload". Please use 'git' or github desktop at least, it is way better for pushing changes and merging conflicts.

File: src/components/NavBar.astro (hunk 2 above)
Comment (txst54) [reply src/components/NavBar.astro]: Also remove the bottom border, should be as similar to the figma file as possible.

**Comment by D3TRU04** [comment]
Screenshot 2024-05-31 at 9 58 55 PM

**Comment by D3TRU04** [comment]
Disregard the footer for now

**Comment by D3TRU04** [comment]
Disregard the footer for now.

**Review by txst54 — APPROVED** [review]
Overall good work. I made your <a></a> thing a component. Try to code efficiently, if you find yourself copying code, make it a component with params passed through props. The reason the font stopped working was because you removed the <link> to the .css file in the Layout. I added it back.

**Pull request merged**